src/solve.jl

```diff
@@ -1,8 +1,9 @@
 function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algType,
               timeseries_init=uType[], ts_init=tType[], ks_init=[];
               d_discontinuities=tType[], dtmax=tType(7*minimum(prob.lags)), dt=zero(tType),
-              kwargs...) where {uType,tType,isinplace,
-                                algType<:AbstractMethodOfStepsAlgorithm,lType}
+              saveat=tType[], tstops=tType[], save_idxs=nothing,
```

Comment:
Yes that's clearly a disadvantage of this approach. Since it only concerns the saving and not the calculations or anything else, I thought it could still be an option. In constrast to ODEs it seems that, at least in general, the ODE integrator really should always save all time steps and all components.